Resources-And-Guides/Scripts/AuthorsNote/inputModifier.js

```diff
@@ -28,7 +28,14 @@ const modifier = (text) => {
         && -1 === authorsNoteIndex
       ) {
         authorsNoteIndex = index
-        state.authorsNote = authorsNoteTokens[1]
+        if (authorsNoteTokens[1].startsWith("-r ")) {
+          state.rawAuthorsNote = true
+          state.authorsNote = authorsNoteTokens[1].substring(3)
```

Comment:
This should be fine, but we might want to consider the case when only "/an -r " is passed. I suppose in that case state.authorsNote would be set to ''.